cmd/kube-apiserver/app/server.go

```diff
@@ -556,7 +565,9 @@ func (s *APIServer) Run(_ []string) error {
 		Authorizer:                authorizer,
 		AdmissionControl:          admissionController,
 		DisableV1:                 disableV1,
-		EnableExp:                 enableExp,
+		EnableExtensions:          enableExtensions,
```

Comment:
It would be really nice if this could be a map/list of groups with a list of versions with a list of resources. We can't keep making variables here for every group/version.